File: prime.js
/*
 * author Cak Juice
 */

function currentMilliSecond() {
	return new Date().getTime();
}

var startTime = currentMilliSecond();

var primes = [];
var limit = 25000;

for (var n=2;n<=limit;n++) {
	var isPrime = true;

	for (var div=2;div<n;div++) {
		if (n % div == 0) {
			isPrime = false;
			break;
		}
	}

	if (isPrime) {
		primes.push(n);
	}
}

var endTime = currentMilliSecond();

console.log(primes);

console.log('Start Time:', startTime);
console.log('End Time:', endTime);
console.log('Selisih:', endTime - startTime, 'ms');

File: prime.go
/*
 * author Cak Juice
 */

package main

import (
	"fmt"
	"time"
)

func main() {
	var startTime int64 = currentMilliSecond()

	var primes []int
	var limit int = 25000

	for n := 2; n <= limit; n++ {
		var isPrime bool = true

		for div := 2; div < n; div++ {
			if n%div == 0 {
				isPrime = false
				break
			}
		}

		if isPrime {
			primes = append(primes, n)
		}
	}

	var endTime int64 = currentMilliSecond()

	fmt.Println(primes)
	fmt.Println("Start Time:", startTime)
	fmt.Println("End Time:", endTime)
	fmt.Println("Selisih:", endTime-startTime, "ms")
}

func currentMilliSecond() int64 {
	now := time.Now()
	return now.UnixNano() / 1000000
}
